ci/run_ci.py

```diff
@@ -0,0 +1,107 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+
+import argparse
+import subprocess
+import platform
+import psutil
+import urllib.request as ulib
+import os
+
+
+def _exec_cmd(cmd: str):
+    print(f"run command: {cmd}")
+    result = subprocess.check_output(cmd, shell=True).decode("utf-8")
+    print(result)
+    return result
+
+
+def _get_os_name_lower():
+    return platform.system().lower()
+
+
+def _get_os_machine():
+    return platform.machine()
+
+
+def _install_bazel():
+    os_name = _get_os_name_lower()
+    if os_name == "windows":
+        # TODO: Implement Windows download bazel.
+        bazel_cmd = "./bazel"
+    else:
+        URL = f"https://github.com/bazelbuild/bazel/releases/download/6.3.2/bazel-6.3.2-installer-{os_name}-{_get_os_machine()}.sh"
```

Comment:
```suggestion
        bazel_version = "6.3.2"
        bazel_download_url = f"https://github.com/bazelbuild/bazel/releases/download/{bazel_version}/bazel-{bazel_version}-installer-{os_name}-{_get_os_machine()}.sh"
```

Reply:
Can be promoted to become a global variable.